### ISSUE TITLE
Certain armies not loading. 

### ISSUE THREAD

**Issue by Thanoontje** [thomascgray/opr-af-to-tts#49]
After patch 3.4, seems some armies break the website when trying to load the list in. So far noticed this with HDF and Watch Brothers.

**Comment by thomascgray** [comment]
Hey, thanks for letting me know about this! Things should be fixed as of https://github.com/thomascgray/opr-af-to-tts/commit/6e77279f45cc54d0dc94be818e2aeaa9e6a517a6#diff-fed4d75f8ae11fb98e76134b44730daee2a90bb6ed422d0ad418755853542d40R520

let me know if you have further problems, cheers!

**Comment by Ixidior** [comment]
Hello there!

the problem persist. I'm using mixed armys. HDF and Crimsom blood (from dakadaka store)

This is the link I'm using.

https://army-forge.onepagerules.com/share?id=4-TRRXl2KipM&name=Bloodiestblood

thanks for your time and work!

**Comment by thomascgray** [comment]
Ah, gotcha. Thanks so much for the link, makes it much easier to test. Will look into this, thank you!

**Comment by Thanoontje** [comment]
First of all thanks for the update :D

Been doing a lil bit of testing after your update, I think I found what is causing the issue. As far as I can tell it at least appears to be with unit's that have mixed toughness across models. Using this list as an example:

https://army-forge.onepagerules.com/share?id=7ImFjAqgLJme&name=Human_Defense%20Force
The list has 2 units. The leaders seem to be fine, but whenever an upgrade is picked that gives a model a weapon team with Tough(3) it breaks the webtool and gives out a blank white page without UI. Making me suspect it has something to do with the tough value being added, as other upgrades seem to be working fine. 

https://army-forge.onepagerules.com/share?id=sl-GsYiIw6em&name=Human_Defense%20Force
This is the one that does seem to be working, as the unit isn't upgraded with a weapon team. 

Thank you for your time and work on this, it's super appreciated!

**Comment by thomascgray** [comment]
OK, all the lists people have posted in this issue are now tested working as of https://github.com/thomascgray/opr-af-to-tts/commit/0937294c8080566fa7a0ffe241b74144fed4af4b

Merged and pushed up, please refresh the site and re-check.

May god forgive me for the level of hacky code I've just produced.

tl;dr a handful of unknown things that have changed with the latest army forge patch. I'm handling a handful of new cases now around data that might be set, not set, located in new places, etc.
